float64[] positions
float64[] velocities
float64[] accelerations
duration time_from_start
